mips1.asm

```diff
@@ -0,0 +1,97 @@
+# Omer Aldafari		
+# Assembly
+# 2/11/15
+# Part 1
+
+# Turn in one .asm file per assignment component
+# Remember to submit it as a pull request to the GitHub repo for the assignment
+
+# fibonacci.asm
+# Input: None
+# Output: Print to terminal N Fibonacci numbers
+
+##########################################################
+# .data segment. Constant and variable definitions go here.
+##########################################################
+
+	.data
+
+fibs:	.word   0 : 9         # Array variable named "fibs" of 9 word-length elements (4 bytes each)
+size:	.word  9              # Single integer variable named "size" that indicates the length of the array
+input: .asciiz "\ Size of the array\n"
+
+###############################################
+# .text segment. Assembly instructions go here.
+###############################################
+	.text
+              la   $a0, input
+              li   $v0, 4
+              syscall
+              
+              li   $v0, 5
+              syscall
+              
+              move $s5, $v0
```

Comment:
-5 Accepting input greater than declared array size is dangerous because your writing to memory that's outside the bounds of the array.